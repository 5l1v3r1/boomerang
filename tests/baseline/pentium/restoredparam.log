Unknown library function proc1
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
in proc twice adding addrExp r28{0} - 4 to local table
finding final parameters for twice
potential param r27
found new parameter r27
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
could not set type for unknown parameter expression m[r28{0} + 4]
could not set type for unknown parameter expression m[r28{0} + 8]
could not set type for unknown parameter expression m[r28{0} + 12]
finding final parameters for twice
