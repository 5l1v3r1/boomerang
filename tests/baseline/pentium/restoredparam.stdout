Boomerang alpha 0.3.1 09/Sep/2006
loading...
decoding entry point...
decoding anything undecoded...
finishing decode...
found 2 procs
decompiling...
 considering main
  considering twice
  decompiling twice
 decompiling main
global type analysis for main
global type analysis for twice
generating code...
output written to ./tests/outputs/pentium/restoredparam
completed in 0 secs.
